From ddc92395b91477834015a6d8617c5b209e13dce0 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:41:55 -0400
Subject: [alsa] hda/realtek: Use alc_write_coef_idx() in alc269_quanta_automake()

Message-id: <140972823573-432-patchset-jkysela@redhat.com>
Patchwork-id: 91128
O-Subject: [RHEL 7.1 PATCH 431/442] ALSA: hda/realtek - Use alc_write_coef_idx() in alc269_quanta_automake()
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Just a refactoring.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>

(picked from kernel: 1687ccc8b2229d05c579924086e9b42ada9db888)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index c9a7a2d..75614e5 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -3355,15 +3355,8 @@ static void alc269_quanta_automute(struct hda_codec *codec)
 {
  snd_hda_gen_update_outputs(codec);
 
- snd_hda_codec_write(codec, 0x20, 0,
-   AC_VERB_SET_COEF_INDEX, 0x0c);
- snd_hda_codec_write(codec, 0x20, 0,
-   AC_VERB_SET_PROC_COEF, 0x680);
-
- snd_hda_codec_write(codec, 0x20, 0,
-   AC_VERB_SET_COEF_INDEX, 0x0c);
- snd_hda_codec_write(codec, 0x20, 0,
-   AC_VERB_SET_PROC_COEF, 0x480);
+ alc_write_coef_idx(codec, 0x0c, 0x680);
+ alc_write_coef_idx(codec, 0x0c, 0x480);
 }
 
 static void alc269_fixup_quanta_mute(struct hda_codec *codec,
-- 
1.7.1